WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/breakpoint/Breakpoint.java

```diff
@@ -1,39 +1,45 @@
 package nl.tudelft.watchdog.core.logic.breakpoint;
 
-import java.io.Serializable;
-
-import com.google.gson.annotations.SerializedName;
-
 /**
  * Data container which stores information about a breakpoint, including its
- *  {@link BreakpointType}.
+ * {@link BreakpointType}.
  */
-public class Breakpoint implements Serializable {
-
-	/** Serial id. */
-	private static final long serialVersionUID = 1L;
+public class Breakpoint {
 
 	/** The hash of this breakpoint. */
-	@SerializedName("bh")
-	private String hash;
-	
+	private int hash;
```

Comment:
It is the hash of the actual breakpoint object passed to the listener's method by the IDE, which is used to identify breakpoints in WD.